iFixit/checkout-ui-extensions
# Add Correct Links For Products To Thank You and Order Status Pages

## Overview

We want to port over some functionality on our thank you/ order status pages.

We want the order summary section to have link to the product pages for the various cart line products

We want there to be a section for product guides if there are any attached to the products in the cart line.

## QA

Make sure the two above cases are working

Closes: https://github.com/iFixit/checkout-ui-extensions/issues/16 https://github.com/iFixit/checkout-ui-extensions/issues/15
